From 81224a85c6c42112a481cbb49a9d35f2b16f4cb9 Mon Sep 17 00:00:00 2001
Message-Id: <81224a85c6c42112a481cbb49a9d35f2b16f4cb9.1349175436.git.minovotn@redhat.com>
In-Reply-To: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
References: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
From: Amos Kong <akong@redhat.com>
Date: Mon, 1 Oct 2012 14:12:43 +0200
Subject: [PATCH 30/34] qerror: drop QERR_SOCKET_CONNECT_IN_PROGRESS

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1349100767-9066-11-git-send-email-akong@redhat.com>
Patchwork-id: 42569
O-Subject: [RHEL-6.4 qemu-kvm PATCH v7 10/14] qerror: drop QERR_SOCKET_CONNECT_IN_PROGRESS
Bugzilla: 680356
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>

From: Luiz Capitulino <lcapitulino@redhat.com>

Unused since last commit.

(Cherry-picked from commit 75b81cf0f26f5bb7ada583fc434835adf03f8b77)

Conflicts:
    qerror.h

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
Reviewed-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Amos Kong <akong@redhat.com>
---
 qerror.c |    4 ----
 qerror.h |    3 ---
 2 files changed, 0 insertions(+), 7 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qerror.c | 4 ----
 qerror.h | 3 ---
 2 files changed, 7 deletions(-)

diff --git a/qerror.c b/qerror.c
index 3e7f410..430692c 100644
--- a/qerror.c
+++ b/qerror.c
@@ -259,10 +259,6 @@ static const QErrorStringTable qerror_table[] = {
         .desc      = "Guest agent command failed, error was '%(message)'",
     },
     {
-        .error_fmt = QERR_SOCKET_CONNECT_IN_PROGRESS,
-        .desc      = "Connection can not be completed immediately",
-    },
-    {
         .error_fmt = QERR_SOCKET_CONNECT_FAILED,
         .desc      = "Failed to connect to socket",
     },
diff --git a/qerror.h b/qerror.h
index 5427664..05a2d0e 100644
--- a/qerror.h
+++ b/qerror.h
@@ -214,9 +214,6 @@ QError *qobject_to_qerror(const QObject *obj);
 #define QERR_QGA_COMMAND_FAILED \
     "{ 'class': 'QgaCommandFailed', 'data': { 'message': %s } }"
 
-#define QERR_SOCKET_CONNECT_IN_PROGRESS \
-    "{ 'class': 'SockConnectInprogress', 'data': {} }"
-
 #define QERR_SOCKET_CONNECT_FAILED \
     "{ 'class': 'SockConnectFailed', 'data': {} }"
 
-- 
1.7.11.4

